tests/integration/012_environment/main.tf

```diff
@@ -76,10 +76,11 @@ resource "env0_team_environment_assignment" "team_role_environment_assignment" {
   role_id        = var.second_run ? env0_custom_role.custom_role1.id : env0_custom_role.custom_role2.id
 }
 
-/* TODO: need to add an integration test.
-resource "env0_environment_state_access" "state_access" {
-  environment_id      = env0_environment.example.id
-  allowed_project_ids = [env0_project.test_project.id]
+/* TODO
```

Comment:
I uncommented this to check that the error went away.
It still fails because of the environment type used (but the original error is now gone).